src/stories/0-Test.stories.tsx

```diff
@@ -1,54 +1,11 @@
-import React, { useContext } from "react";
-import { addDecorator } from "@storybook/react";
+import React from "react";
 import FracTable from "../components/FracHitTable";
-import { Provider, useDispatch } from "react-redux";
-import { store, frachitActions, AppDispatch } from "../store";
+import { withProvider } from '../../.storybook/decorators'
 
 export default {
   title: "FrachitPanel",
   component: FracTable,
+  decorators: [withProvider],
 };
 
-export const FracHitTable = () => {
-  const dispatch: AppDispatch = useDispatch();
-  /*
-  dispatch(
-    frachitActions.addHitThunk([
-      0,
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      459.45,
-      1,
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      "",
-    ])
-  ); 
-  dispatch(
-    frachitActions.addHitThunk([
-      1,
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      469.45,
-      1,
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      "",
-    ])
-  );
-  dispatch(
-    frachitActions.addHitThunk([
-      2,
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      479.45,
-      1,
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      new Date(2020, 11, 24, 10, 33, 30).getTime(),
-      "",
-    ])
-  );*/
-
-  return (
-    <div style={{ display: "flex" }}>
-      <FracTable />
-    </div>
-  );
-};
+export const FracHitTable = () => <FracTable />
```

Comment:
It seems like to me to initialize different displays, you'd initialize the store with those values so the selector will work as expected. Most examples show passing in props, but i think for `useSelector` to work you'd have to initialize the store. That being said, you might need to reconfigure how your store creation works, but not 100% sure without seeing a full project.